openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedProvider/ActivityFeedProvider.tsx

```diff
@@ -116,7 +116,7 @@ const ActivityFeedProvider = ({ children, user }: Props) => {
 
         const { data, paging } = await getAllFeeds(
           entityType !== EntityType.USER_NAME
-            ? getEntityFeedLink(entityType, fqn)
+            ? getEntityFeedLink(entityType, encodeURIComponent(fqn ?? ''))
```

Comment:
What's point of encoding and empty string?

If fqn is not there then link doesn't make any sense right?